indra/llcharacter/llcharacter.cpp

```diff
@@ -65,7 +65,9 @@ LLCharacter::LLCharacter()
 //-----------------------------------------------------------------------------
 LLCharacter::~LLCharacter()
 {
-    for (const auto& it : mVisualParamIndexMap)
+    visual_param_index_map_t local_map;
+    std::swap(local_map, mVisualParamIndexMap);
+    for (const auto& it : local_map)
```

Comment:
Please add a meaningfull comment, why is this needed in destrucor? Fix implies that visual param destructor can alter mVisualParamIndexMap, but that doesn't appear to be true. And I see even less relation to the crash report.